### ISSUE TITLE
failed to activate under web proxy environment

### ISSUE THREAD

**Issue by utotch** [mathworks-ref-arch/matlab-integration-for-jupyter#10]
I tried to use matlab/Dockerfile image.
I succeeded to make the image and run it, and launched "Open Matlab", which showed Online License Manager page successfully.

But even though I entered correct e-mail and password, the page failed to activate my license using web proxy environment.
When I entered the correct password and push "Sign in" button, the license manager showed any changes.

I tried to set web proxy related environment variables
* http_proxy
* https_proxy
* HTTP_PROXY
* HTTPS_PROXY
but they didn't help to solve the problem.

I suffered from an activation problem of Matlab on Linux under proxy many times.
I hope Mathworks carefully test and implement the authentication scheme under a proxy environment on LInux.

**Comment by krisctl** [comment]
Thanks for trying out matlab integration for jupyter, @utotch!

I understand the login screen is unresponsive when you click Sign in after entering the correct credentials. Kindly confirm that you have recently built this image (after August 7, 2023) because a related fix was released in [matlab-proxy](https://github.com/mathworks/matlab-proxy/releases/tag/v0.7.3) project on Aug 7.

Secondly, can you verify if there is any error displayed on the terminal window from where you are running this docker image? It would help us immensely if you could share those logs with us as well.

**Comment by utotch** [comment]
Thanks for the comment.
I used the ratest version of the Dockerfile but it failed to activate under a web proxy.

my env is below:
```
$ pip freeze | grep matlab
jupyter-matlab-proxy==0.6.1
matlab-proxy==0.7.3
matlabengineforpython==9.14
```

I set
* http_proxy
* https_proxy
* HTTP_PROXY
* HTTPS_PROXY
* no_proxy
* NO_PROXY

correctly, but the online license manager couldn't log in under the web proxy.
And there was no error or information displayed on the terminal window.
(It showed a Matlab is correctly started but no response when I operated the license manager.)

I hope this helps to solve the problem.

**Comment by prabhakk-mw** [comment]
@utotch 
Could you please verify that you get this error with the `mathworks/matlab` containers on dockerhub as well?

To pull the R2023b MATLAB image to your machine, execute:
```
docker pull mathworks/matlab:r2023b
```

To launch the container with the `-browser` option, execute:
```
docker run -it --rm -p 8888:8888 --shm-size=512M mathworks/matlab:r2023b -browser
```

**Comment by utotch** [comment]
Dear @prabhakk-mw,
Sorry for this late reply.

I tried the process above, and the response is precisely the same as I encountered by using jupyter container.

The weird thing is that when I input an incorrect password, the online manager responds, "Invalid Email or Password", but when I input the correct password, it responds nothing and just doesn't proceed to the next.

Thus the online manager looked like it recognized the correct password.
